lab.rb

```diff
@@ -0,0 +1,134 @@
+# require 'pry'
+
+class Flashcard
+
+  attr_accessor :number, :front, :back
+  def initialize number, front, back
+    @number = number
+    @front = front
+    @back = back
+
+  end
+
+end
+
+class Game
+  attr_accessor :answers
+def initialize
+  @cards = []
+  @answers = []
+  @score = 0
+  @questions = 0
+end
+
+def new_flash_card
+  puts "enter info for front"
+  front_card = gets.chomp
+  puts "enter info for back of card"
+  back_card = gets.chomp
+  number = @cards.length + 1
+  new_card = Flashcard.new(number, front_card, back_card)
+  @cards << new_card
+end
+
+def display_card
+  @cards.each do |card|
+  puts "#{card.number} - front: #{card.front}, back: #{card.back}"
+end
+end
+
+def edit_card
+  puts "enter number of card you would like to edit"
+  card_num = gets.chomp.to_i
+  selected_card = @cards.find do |card|
+    card.number == card_num
+  end
+  puts "Edit card front: "
+  new_front = gets.chomp
+
+  puts "Edit card back: "
+  new_back = gets.chomp
+
+  selected_card.front = new_front
+  selected_card.back = new_back
+end
+
+def delete_card
+  puts "enter number of card you would like to delete"
+  card_num = gets.chomp.to_i
+  selected_card = @cards.find do |card|
+    card.number == card_num
+  end
+   @cards -= [selected_card]
+end
+def play_game
+  puts "rules: you will be prompted to "
+  puts "enter the word that matches the definition given"
+  puts "yes, like jeopordy but without the what is part"
+  @cards.each_with_index do |l|
+    puts "#{l.back}"
+    input = gets.chomp
+    @answers << input
+
+    until input == l.front
+      @questions += 1
+      puts "wrong! you've gotten #{@score} out of #{@questions} right, TRY AGAIN"
+      input = gets.chomp
+      @answers << input
+    if input == l.front
+    @score += 1
+    @questions += 1
+    puts "right! you've gotten #{@score} out of #{@questions} right"
+end
```

Comment:
Any time you see 2 or more `end` statements all pushed up against the edge of the page like this, there's some indentation to be fixed above.